### PR TITLE
docs(self-managed): amazon eks support

### PR DESCRIPTION
**Pull request by aabouzaid**
## What is the purpose of the change

Documenting Amazon EKS support in Camunda Platform 8 Self-Managed.

## Are there related marketing activities

No.

## When should this change go live?

As soon as possible.

## PR Checklist

- [x] My changes are not for an already released version.
- [x] My changes apply to future versions, and I have added them to the main `/docs` directory.
- [x] My changes do not require an Engineering review.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @christinaausley as a reviewer.

### CODE REVIEW

**Comment by aabouzaid** [comment]
The error shows that the `known-limitations` page is not found, and that's expected because it's renamed to `troubleshooting`.

File: docs/self-managed/platform-deployment/amazon-eks.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
+---
+id: amazon-eks
+title: "Amazon EKS"
+---
+
+Amazon Elastic Kubernetes Service ([Amazon EKS](https://aws.amazon.com/eks/)) is a managed
+container service to run and scale Kubernetes applications in the cloud or on-premises.
```
Comment (christinaausley) [inline docs/self-managed/platform-deployment/amazon-eks.md:7]: instead of on-premises, do you mean Self-Managed here?

**Review by christinaausley — APPROVED** [review]
Thank you for these changes! I had one comment for clarification, and made some stylistic adjustments in my latest commit, including adding descriptions in the headers.

File: docs/self-managed/platform-deployment/amazon-eks.md (hunk 1 above)
Comment (aabouzaid) [reply docs/self-managed/platform-deployment/amazon-eks.md:9]: nope, here I'm referring to Amazon EKS functionally, not Camunda Platform. (it's actually how Amazon EKS is described on its [home page](https://aws.amazon.com/eks/)).

File: docs/self-managed/platform-deployment/troubleshooting.md [hunk 2]
```diff
@@ -2,26 +2,25 @@
...
-for advice, we would recommend viewing the solution available in the
+not provide support for the distribution of the Keycloak TLS key to the other containers, we recommend viewing the solution available in the
 [Identity documentation](/docs/self-managed/identity/troubleshooting/common-problems#solution-2-identity-making-requests-from-an-external-ip-address).
 
 ## Identity redirect URL
 
-If HTTP to HTTPS redirection is enabled in the Load-balancer or Ingress, make sure to use the HTTPS
-protocol in the values file under `global.identity.auth.[COMPONENT].redirectUrl'.
-Otherwise you will get a redirection error in Keycloak.
+If HTTP to HTTPS redirection is enabled in the load-balancer or ingress, make sure to use the HTTPS
```
Comment (aabouzaid) [inline docs/self-managed/platform-deployment/troubleshooting.md:21]: The `Ingress` word here is a proper name. (refers to [Kubernetes Ingress](https://kubernetes.io/docs/concepts/services-networking/ingress/)).

**Comment by aabouzaid** [comment]
@christinaausley thank you for your review :rocket: 
I've answered your question and left a tiny comment about `Ingress`.

File: docs/self-managed/platform-deployment/amazon-eks.md [hunk 3]
```diff
@@ -1,13 +1,14 @@
 ---
 id: amazon-eks
 title: "Amazon EKS"
+description: "Amazon EKS is a managed
```
Comment (aabouzaid) [inline docs/self-managed/platform-deployment/amazon-eks.md:4]: I'm not sure about the convention we use for the description, but I believe it should be something around C8SM and EKS like:

```
description: "Deploy Camunda Platform 8 Self-Managed on Amazon EKS"
```

Comment (christinaausley) [reply docs/self-managed/platform-deployment/amazon-eks.md]: Agreed here. Updated in latest commit.

**Review by christinaausley — APPROVED** [review]

**Comment by akeller** [comment]
@pepopowitz added a redirect/rewrite rule for this as part of our docs release process for 8.1/3.9.0.

**Pull request merged**